controllers/dashboard.js

```diff
@@ -126,7 +126,7 @@ router.get('/', (req, res, next) => {
 				},
 				{
 					dueDate: {
-						$gte: new Date().getTime(),
+						$gte: ((new Date().getTime()) - 1000 * 60 * 60 * 24 * 7), // homeworks with max. 7 days after dueDate
```

Comment:
Kommentar und logik passen igw. nicht zusammen (oder ich bin zu doof)
Wir suchen "HA mit Abgabedatum größer als heute vor 7 Tagen", bedeutet doch `// homeworks with max. 7 days before dueDate`
```suggestion
						$gte: ((new Date().getTime()) - 1000 * 60 * 60 * 24 * 7), // homeworks with max. 7 days before dueDate
```

Reply:
Das ist richtig, dass meinte ich auch :)